mm2src/mm2_main/src/rpc.rs

```diff
@@ -178,35 +175,6 @@ fn response_from_dispatcher_error(
     response.serialize_http_response()
 }
 
-pub fn escape_answer<'a, S: Into<Cow<'a, str>>>(input: S) -> Cow<'a, str> {
-    lazy_static! {
-        static ref REGEX: Regex = Regex::new("[<>&]").unwrap();
-    }
-
-    let input = input.into();
-    let mut last_match = 0;
-
-    if REGEX.is_match(&input) {
-        let matches = REGEX.find_iter(&input);
-        let mut output = String::with_capacity(input.len());
-        for mat in matches {
-            let (begin, end) = (mat.start(), mat.end());
-            output.push_str(&input[last_match..begin]);
-            match &input[begin..end] {
-                "<" => output.push_str("&lt;"),
-                ">" => output.push_str("&gt;"),
-                "&" => output.push_str("&amp;"),
-                _ => unreachable!(),
-            }
-            last_match = end;
-        }
-        output.push_str(&input[last_match..]);
-        Cow::Owned(output)
-    } else {
-        input
-    }
-}
```

Comment:
Any idea what was the motivation for this?

Reply:
context: https://matrix.to/#/!yeLsKrHmWqEMEAILHs:matrix.org/$mAxamnv9Ga8rxAg24mXAUwRu9bKQArw2tB_aNaA177c?via=matrix.org